README.md

```diff
@@ -10,33 +10,59 @@ Obelisk should be used as a dependency for external robot control code that is w
-* The `--cyclone-perf` flag adds [performance optimizations for Cyclone DDS](https://github.com/ros2/rmw_cyclonedds?tab=readme-ov-file#performance-recommendations) in the `/etc/sysctl.d/60-cyclonedds.conf` file on your local filesystem. You should  specify this if you plan to use Obelisk in a non-containerized environment.
-* The `--bash-aliases` flag will check if `~/.bash_aliases` is sourced in the `~/.bashrc` file (and will add it if not already in there), and will create the `~/.bash_aliases` file if it doesn't already exist. This is a very benign flag, so we recommend using it.
-* The `--obk-aliases` flag will add useful Obelisk aliases to the `~/.bash_aliases` file. **We highly recommend using this flag.**
-* If you trust us, you can use the `--all` flag to just opt-in to all of these dependencies.
-If you're more cautious, we recommend running
+Initial setup proceeds by running the `setup.sh` script in the repository root. This script has the ability to make changes to your local dependencies - all such changes are opt-in. **It is very important that you run `setup.sh` using the `source` command, and not `bash`, because there are environment variables that will be sourced!**
+
+This script has the ability to do 3 things:
+1. configure a conditional Docker build so that the image has the right dependencies in it to run Obelisk
+2. install system dependencies **on the machine running this script**
+3. set up user-specific settings, including very useful bash aliases
+
+The options are as follows:
 ```
-source setup.sh --pixi --bash-aliases --obk-aliases
+source setup.sh [OPTIONS]
+
+Options:
+  --recommended                Apply recommended system-level changes
+                               (cyclone performance optimizations, pixi, obelisk aliases)
+
+  --basic                      Enables basic dependencies necessary for Obelisk locally
+  --cyclone-perf               Enables cyclone performance optimizations
+  --leap                       Enables LEAP hand dependencies
+  --zed                        Enables ZED SDK
+
+  --docker-install             Install Docker and nvidia-container-toolkit
+  --install-sys-deps-docker    Installs system dependencies in Docker
+
+  --install-sys-deps           Installs system dependencies
+  --source-ros                 Sources base ROS in ~/.bashrc (only used if --install-sys-deps)
+
+  --pixi                       Install pixi
+  --obk-aliases                Add obelisk aliases to the ~/.bash_aliases file
+
+  --help                       Display this help message and exit
 ```
+
+Some guidance/recommendations on choosing flags:
+* If you don't have Docker on your machine, use the `--docker-install` flag
+* If you are not using Docker, then you should use `--install-sys-deps`
+* If you are using Docker, but not pixi, you should also use `--install-sys-deps`
+* If you are using Docker, use `--install-sys-deps-docker` if and only if you are **not** using pixi within the container.
```

Comment:
Reading these two lines is a bit confusing. Can we clarify this a bit. Right now both conditions read the same, but suggest doing different things. I.e. both suggest something different for docker with no pixi.

Reply:
good catch, there was a typo I believe